flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64"]
```

Comment:
no build from source to support aarch64?

Reply:
Maybe. But from what I have seen in the electron build guide it's a major hassle. But won't put it off the table

Reply:
since there is an upstream aarch64 package it can work too. Just that usually there is none.